sisyphus/localengine.py

```diff
@@ -297,7 +316,7 @@ def try_to_recover_task(self, task, task_id):
 
             with self.running_tasks as running_tasks:
                 name = (task_instance.name, task_id)
-                running_tasks[name] = (process, task_instance)
+                running_tasks[name] = (process, task_instance, "")
```

Comment:
You could try to read it from `/proc/$pid/environ`. I don't know of any nicer interface to that information...
Maybe the current `CUDA_VISIBLE_DEVICES` could be added as part of the usage file. This would be a useful information to have in other cases as well.